src/utils/errorMessages.ts

```diff
@@ -0,0 +1,20 @@
+// errorMessages.ts
+
+// Error message mapping for different error signatures
+const errorMapping: Record<string, string> = {
+    // 'TooManyActiveOrders' corresponds to the error signature '0x28d7a641'
+    // error TooManyActiveOrders();
+    '0x28d7a641': 'TooManyActiveOrders',
+    // More mappings can be added here...
+};
```

Comment:
With this way, we need to add all errors to be supported like this.
Is there any way to attach the library's ABI, like the following unit test using hardhat and ethers?

https://github.com/Secured-Finance/contracts/blob/4ba147ea4d7b617d61176930497495185d39c535/test/performance/order-book.test.ts#L101